### PR TITLE
modification of aliasPay request schema

### PR DESCRIPTION
**Pull request by cortesa**

### CODE REVIEW

File: server/nodejs/api/swagger.yaml [hunk 1]
```diff
@@ -5045,22 +5045,27 @@ definitions:
...
           :amount, ... } (numeric or string) The syscoin alias is the key, the numeric\
           \ amount (can be string) in SYS is the value."
+        properties: {}
       instantsend:
         type: "boolean"
         description: "Set to true to use InstantSend to send this transaction or false\
           \ otherwise."
       subtractfeefromamount:
-        type: "string"
+        type: "array"
```
Comment (justicegray) [inline server/nodejs/api/swagger.yaml]: @quangogster @cortesa If my assumption is correct about why subtractfeefromamount is an array and not a string, we might benefit from the amounts object carrying that payload in a future release.  e.g. an "amount" has address, amount, subtractfeefromamount.  Nothing to do for this release if this works but I wouldn't mind revisiting.

File: server/nodejs/api/swagger.yaml [hunk 2]
```diff
@@ -5045,22 +5045,27 @@ definitions:
...
         description: "Set to true to subtract fee from amount or false otherwise."
+        items:
+          type: "string"
     example:
-      amounts: "amounts"
+      amounts: "{}"
```
Comment (justicegray) [inline server/nodejs/api/swagger.yaml]: Could we provide a slightly more detailed object example? =)

Comment (cortesa) [reply server/nodejs/api/swagger.yaml]: Examples added

File: server/nodejs/api/swagger.yaml [hunk 3]
```diff
@@ -5045,22 +5045,27 @@ definitions:
         type: "string"
         description: "Alias to pay from"
       amounts:
-        type: "string"
-        description: "A stringified JSON object with aliases and amounts. { \"address\"\
-          :amount, ... } (numeric or string) The syscoin alias is the key, the numeric\
-          \ amount (can be string) in SYS is the value."
+        $ref: "#/definitions/AliasPayRequest_amounts"
```
Comment (justicegray) [inline server/nodejs/api/swagger.yaml]: Actually can we change this object to AliasPayRequestAmounts?

Comment (cortesa) [reply server/nodejs/api/swagger.yaml]: it is auto generated from the `./gen_server_yaml.sh`

Comment (cortesa) [reply server/nodejs/api/swagger.yaml]: it is as you said in swagger.yaml in the root folder.

**Pull request merged**